input/fsh/AL_CodeSystems.fsh

```diff
@@ -20,8 +20,8 @@ Alias:   VerStatus = http://terminology.hl7.org/CodeSystem/condition-ver-status
 
 // Local
 // Codesystems
-Alias: $SCT_TBD = http://snomed.info/sct // SnomedRequestedCS
 Alias: $CodexRTCategoryCS = http://snomed.info/sct //$SCT_TBD // 
+Alias: $SCT = http://snomed.info/sct
```

Comment:
Now we have SCT and $SCT. Should we use only one of them?

Reply:
Thanks for catching my sloppiness....fixed